man/systemd-tmpfiles.xml

```diff
@@ -227,6 +230,24 @@
     <programlisting>systemd-tmpfiles --remove --create</programlisting>
   </refsect1>
 
+  <refsect1>
+    <title>Environment</title>
+
+    <variablelist class='environment-variables'>
+      <xi:include href="common-variables.xml" xpointer="log-level" />
+      <xi:include href="common-variables.xml" xpointer="log-color" />
+      <xi:include href="common-variables.xml" xpointer="log-time" />
+      <xi:include href="common-variables.xml" xpointer="log-location" />
+      <xi:include href="common-variables.xml" xpointer="log-target" />
+      <xi:include href="common-variables.xml" xpointer="pager" />
+      <xi:include href="common-variables.xml" xpointer="less" />
+      <xi:include href="common-variables.xml" xpointer="lesscharset" />
+      <xi:include href="common-variables.xml" xpointer="lesssecure" />
+      <xi:include href="common-variables.xml" xpointer="colors" />
+      <xi:include href="common-variables.xml" xpointer="urlify" />
+    </variablelist>
```

Comment:
i must say, I am not a a fan of embedding these into each and every man page we have. Maybe instead have a single reference to the "Common variables for all systemd command line tools" or so and then have a separate man page for that. I think there's value in having short, non-redundant man pages.

I'd thus suggest:

1. have a proper man page that lists the "common" env vars, and ref that from all our tool man pages
2. have the existing markdown document for the "exotic" developer/hacker/debug env vars, and ref that from the "common" env vars man page
3. Drop all lists of env vars from all our man pages otherwise – except if there's a tool-specific env var that is "common" and not "exotic — though in that case it probably shouldn't be an env var in the first place, but some cmdline arg or so.

Reply:
I thought about this approach too, but the problem is that each tool supports a slightly different set of variables (or sometimes, some variables are supported but don't make any sense). So we would still need some way to narrow down which variables are useful in a given tool.